images/kubeadm-dind/kubeadm_dind_runner

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Enable IPv6 for Kubeadm DinD Cluster based CI
+sysctl -w net.ipv6.conf.all.disable_ipv6=0
```

Comment:
hmm do we need another image just to do this?

Reply:
If it's okay to enable IPv6 in kubekins-e2e, then we should be able to use that image directly.

Reply:
What if we conditionally enable it in kubekins-e2e instead?

Reply:
This would be enabled based upon an environment variable that we'd set in prow/config.yaml? (That is, rather than it being based upon the --ip-mode=ipv6 config in jobs/config.json... since that flag would have to be interpreted in bootstrap.py or scenario/kubernetes_e2e.py). Something like:
      env:                                              
       - name: IPV6_ENABLE
         value: "true"

Reply:
@BenTheElder - How does this look: PR #9102 ?
I figured this IPv6-enable change might make sense to do separately so that bootstrap and kubekins-e2e containers using it can be built and pushed ahead of this PR 7529.

Reply:
@BenTheElder - the suggestion for PR #9102 is to move this enabling of IPv6 to kubetest (rather than offer it in the bootstrap runner, since bootstrap container is generic). Optionally, I can move this to kubernetes_e2e.py or to the kubeadm_dind deployment (difference being how generally available the IPv6 enabling is to other CI test cases). Do you have a preference where I should put this?